blazor/treegrid/events.md

```diff
@@ -2453,7 +2453,7 @@ The [DetailDataBound](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor~Sy
 
 ## RowDropping
 
-[RowDropping](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor~Syncfusion.Blazor.TreeGrid.TreeGridEvents%601~RowDropping.html) event triggers when the row elements are dropped on the target element.
+[RowDropping](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.RowDroppingEventArgs-1.html) event triggers when the row elements are dropped on the target element.
```

Comment:
Seems you refereed the Grid event URL here. Need to update TreeGrid Row Dropping event URL here.